client/dist/styles.css

```diff
@@ -92,3 +111,46 @@ button {
   display: flex;
   flex-direction: column;
 }
+.profile-avatar {
+  width: 50px;
+  height: 50px;
+  float: left;
+}
+
+.profile-name {
+  font-weight: bold;
+}
+.user-info {
+
+}
+.pay-button-container {
+  padding: 10px;
+  background-color: #eee;
+  border: 1px solid #ccc;
+}
+
+.payment-container {
+  width: 100%;
+  padding: 20px;
+  position: 'relative';
+  margin: 20px;
+
+}
+.payment-item-container {
+  display: 'block';
+  justify-content: 'space-evenly';
+}
+.form-box {
+  padding-bottom: 20px;
+}
+.bottom {
+}
+.top {
```

Comment:
empty class.

Reply:
I think we might want to use more specific classnames than "top, left, right'. Thoughts?